packages/flutter_tools/test/commands.shard/hermetic/config_test.dart

```diff
@@ -228,71 +228,38 @@ void main() {
     );
 
     testUsingContext(
```

Comment:
Explanation for this part of the diff:

* Actually make sure `Analytics` is enabled/disabled by `flutter config --analytics`
* Remove check for enabled/disabled event. In package:unified_analytics, this event bypasses the `Analytics.send` method, so it's not recorded to `FakeAnalytics.sentEvents`.